tests/unit/ui/components/TestDropdownMenuItem.js

```diff
@@ -0,0 +1,71 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import DropdownMenuItem from 'ui/components/DropdownMenu/DropdownMenuItem';
+import Link from 'amo/components/Link';
+
+
+describe(__filename, () => {
+  it('renders a section when only `title` is specified', () => {
+    const item = shallow(<DropdownMenuItem title="A section" />);
```

Comment:
The `title` attribute here seems like it would be better suited to just being the `children` attribute. It's not rendering a `title` attribute which is a common HTML attribute, so re-using that here for what is effectively `children` (these get rendered as text inside the button/link components) is confusing.

Reply:
Oh that makes a lot of sense!